bin/gen_ide_helper_models.php

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+require_once __DIR__ . '/../includes/application.php';
+
+/**
+ * This script generates the models helper file "_ide_helper_models.php" into the project root.
+ */
+
+$config = $app->get('config');
+$config['ide-helper.model_locations'] = ['src/Models'];
+
+$fileSystem = new \Illuminate\Filesystem\Filesystem();
+$command = new \Barryvdh\LaravelIdeHelper\Console\ModelsCommand($fileSystem);
+$command->setLaravel($app);
+$command->run(
+    new \Symfony\Component\Console\Input\ArrayInput([
+        '--nowrite' => true,
```

Comment:
Is there a reason why it should not be written directly?

Reply:
Yes like you found out in the other PR. This means writing to the model files. But then also stuff like the id property will be written. The idea here is: use it for generation to a file where you can grab the parts you need from.

Reply:
Well, that makes sense :D